README.md

```diff
@@ -1,35 +1,20 @@
-# OSM_DataManipulation
+#Sateltiles
 ##Prerequisities
--Python 2.7
+ -Python 2.7
+ 
+ -wxPython(GUI plateform)
+ 
+ -Psycopg2(PostgreSQL database adapter)
+ 
+##Installation
+ -Install all prerequisities.
+ 
+ 
```

Comment:
don't leave extra empty lines. a single blank line at end is okey for parsing not more